ts/components/search/SearchErrorMessage.tsx

```diff
@@ -0,0 +1,73 @@
+import { Text, View } from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import I18n from "../../i18n";
+import customVariables from "../../theme/variables";
+
+const styles = StyleSheet.create({
+  contentWrapper: {
+    flex: 1,
+    alignItems: "center"
+  },
+  message: {
+    fontSize: customVariables.fontSizeBase,
+    padding: customVariables.contentPadding,
+    textAlign: "center"
+  }
+});
+
+export enum ErrorSearchType {
+  InvalidSearchBarText,
+  NoResultsFound
+}
+
+/**
+ * Print the icon according to current error type
+ * @param errorType
+ */
+export const renderIconErrorSearch = (errorType: ErrorSearchType) => {
+  switch (errorType) {
+    case ErrorSearchType.InvalidSearchBarText:
+      return require("../../../img/search/search-icon.png");
+    case ErrorSearchType.NoResultsFound:
+      return require("../../../img/search/beer-mug.png");
+  }
+};
+
+/**
+ * Convert the error type into a user-readable string
+ * @param errorType
+ */
+export const renderMessageErrorSearch = (
+  errorType: ErrorSearchType
+): string => {
+  switch (errorType) {
+    case ErrorSearchType.InvalidSearchBarText:
+      return I18n.t("global.search.invalidSearchBarText");
+    case ErrorSearchType.NoResultsFound:
+      return I18n.t("global.search.noResultsFound");
+  }
+};
+
+type Props = {
+  errorType: ErrorSearchType;
+};
+
+export class SearchErrorMessage extends React.PureComponent<Props> {
```

Comment:
This is only a suggestion: what do you think about naming `SearchNoResultMessage` instead of `SearchErrorMessage`?
I guess this component is responsible to show a message when there are no results to show. The reasons can be: no valid input or no results found. 
```suggestion
export class SearchNoResultMessage extends React.PureComponent<Props> {
```

Reply:
@Undermaken I called it that because maybe it could be used in the future to render some other kind of search error. But yes, at the moment it deals strictly with these 2 types of errors and we could rename it as you suggest.

Reply:
Done. In the commit f3a51b5